testing/infra/terraform/modules/ec_deployment/variables.tf

```diff
@@ -123,3 +123,17 @@ variable "apm_index_shards" {
   description = "The number of shards to set for APM Indices"
   type        = number
 }
+
+# Install custom APM integration package
+
+variable "install_custom_apm_integration_pkg" {
+  type = bool
+  description = "Whether or not to install a custom APM integration package"
+  default = false
+}
```

Comment:
Can we infer this from `custom_apm_integration_pkg_path` being non-empty? Is there any reason to specify one without the other?

Reply:
I wanted to be more explicit here since not all users of `ec_deployment` module will want to install custom integration package. I have fixed this to use only 1 variable and updated the description of the variable to notify users that empty string will not install a custom integration.

Reply:
Got it. If it creates confusion then we can revisit, but IMO the name and description are (now) sufficiently self-explanatory.

Reply:
Makes sense, I have updated it to use a single variable.